docs/scenarios/report/README.md

```diff
@@ -0,0 +1,41 @@
+## Generating pre-migration report with CPMA
+
+---
+
+### 1. Prerequisites
+
+Prior to working with CPMA you need to deploy an OCP3 cluster(OCP 3.7, 3.9, 3.10, 3.11 should be supported).
+In order to generate a report, CPMA interacts with OCP using it's API. This means KUBECONFIG is required as a well as user used to talk with OCP cluster API needs to have priviligies to list nodes, projects, pods etc. Recommended cluster role is `system:admin`. It can be configured with following command: `oc adm policy add-cluster-role-to-user system:admin <username>`.
+
+---
+
+### 2. General CPMA configuration
+
+CPMA can be configured in a couple different ways.
```

Comment:
"CPMA can be configured using either:"
I understand a couple of" means two or more" but that's ambiguous for lots of international English speakers (or readers), my 2cents, just to avoid confusion.